sw/device/tests/crypto/cryptotest/json/ibex_sca_commands.h

```diff
@@ -0,0 +1,46 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_IBEX_SCA_COMMANDS_H_
```

Comment:
Are we missing the required target in the BUILD file? 

This may be the reason why we see this CI error: 

```
no such target '//sw/device/tests/crypto/cryptotest/json:ibex_sca_commands': target
```

Reply:
Ah thanks for pointing this out. Fixed the issue.